File: api/src/main/java/fr/huiitre/tools/modules/riot/valorant/application/user/view/ValorantStoreHistoryView.java
package fr.huiitre.tools.modules.riot.valorant.application.user.view;

import fr.huiitre.tools.modules.riot.valorant.application.skin.view.ValorantSkinView;
import java.time.LocalDate;
import java.util.List;

public record ValorantStoreHistoryView(
    LocalDate date,
    List<ValorantSkinView> skins
) {}

File: api/Modules/Riot/Valorant/Application/User/Views/ValorantStoreHistoryView.cs
using Tools.Api.Modules.Riot.Valorant.Application.Skin.Views;

namespace Tools.Api.Modules.Riot.Valorant.Application.User.Views;

public sealed record ValorantStoreHistoryView(
    DateOnly Date,
    List<ValorantSkinView> Skins
);
